### ISSUE TITLE
Add a nice Display method for pc group

### ISSUE THREAD

**Issue by fingolfin** [gap-system/gap#3538]
It would be nice if there was an easy way to "see" the pc relation of a pc group. My suggestion would to add a `Display` (and/or `DisplayString`) method to do the job. Somewhat similar to what the `polycyclic` package offers with the `PrintPcpPresentation` command.

To illustrate what I mean, compare this:
```
gap> g:=SymmetricGroup(IsPcGroup, 4);
<pc group of size 24 with 4 generators>
gap> Display(g);
Group( [ f1, f2, f3, f4 ] )
```

with this:
```
gap> h:=SymmetricGroup(IsPcpGroup, 4);
Pcp-group with orders [ 2, 3, 2, 2 ]
gap> PrintPcpPresentation(h);
g1^2 = id
g2^3 = id
g3^2 = id
g4^2 = id
g2 ^ g1 = g2^2
g3 ^ g1 = g4
g3 ^ g2 = g4
g4 ^ g1 = g3
g4 ^ g2 = g3 * g4
```

Here is a suggestion what the display output could look like:
```
gap> Display(g)
pc-group with relations:
  g1^2 = id
  g2^3 = id
  g3^2 = id
  g4^2 = id
  g2 ^ g1 = g2^2
  g3 ^ g1 = g4
  g3 ^ g2 = g4
  g4 ^ g1 = g3
  g4 ^ g2 = g3 * g4
```

(Incidentally, I think that `polycylic` also should have custom `Display` method which shows the presentation).

**Comment by wucas** [comment]
Can this be closed since #3621 git merged?

**Comment by DominikBernhardt** [comment]
I'll close this as #3621 is merged.
